src/libraries/Native/Unix/System.Native/pal_iosversion.h

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "pal_compiler.h"
+#include "pal_types.h"
+
+PALEXPORT const char* SystemNative_iOSSupportVersion();
```

Comment:
the function needs to be added to https://github.com/dotnet/runtime/blob/main/src/libraries/Native/Unix/System.Native/entrypoints.c